xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/XWiki.java

```diff
@@ -3396,7 +3396,12 @@ public int createUser(boolean withValidation, String userRights, XWikiContext co
             // Send validation mail, if needed.
             if ((result > 0) && (withValidation)) {
                 // Send the validation email
-                sendValidationEmail(xwikiname, password, email, validkey, "validation_email_content", context);
+                try {
+                    sendValidationEmail(xwikiname, password, email, validkey, "validation_email_content", context);  
+                } catch (XWikiException e) {
+                    LOGGER.error("User created. Failed to send the email to the created user", e);
+                    return -11;
```

Comment:
Note that this look too magical and it's duplicated in 2 places. Better introduce a constant.